src/docs/exampleComponents/ComposedChart/BandedChart.js

```diff
@@ -0,0 +1,83 @@
+import React, { PureComponent } from 'react';
+import {
+  ComposedChart,
+  Line,
+  Area,
+  XAxis,
+  YAxis,
+  CartesianGrid,
+  Tooltip,
+  ResponsiveContainer,
+} from 'recharts';
+
+const data = [
+    {
+      name: "Page A",
+      a: [0, 0],
+      b: 0,
+    },
+    {
+      name: "Page B",
+      a: [50, 300],
+      b: 106,
+    },
+    {
+      name: "Page C",
+      a: [150, 423],
+    },
+    {
+      name: "Page D",
+      b: 312,
+    },
+    {
+      name: "Page E",
+      a: [367, 678],
+      b: 451,
+    },
+    {
+      name: "Page F",
+      a: [305, 821],
+      b: 623,
+    },
+  ];
+  
+  export default class Example extends PureComponent {
+    renderTooltipWithoutRange = ({ payload, content, ...rest }) => {
+        const newPayload = payload.filter((x) => x.dataKey !== "a");
+        return <Tooltip payload={newPayload} {...rest} />;
+    }
+
+    render() {
+        return (
+          <ResponsiveContainer width="100%" height="100%">
+            <ComposedChart
+            width={500}
+            height={400}
+            data={data}
+            margin={{
+                top: 10,
+                right: 30,
+                left: 0,
+                bottom: 0,
+            }}
+            >
+            <CartesianGrid strokeDasharray="3 3" />
+            <XAxis dataKey="name" />
+            <YAxis />
+            <Tooltip content={this.renderTooltipWithoutRange} />
```

Comment:
Trying to repeat the same trick to `<Legend>` element gives me weird result when Legend's block is shifted and overlaps chart's area. Does not happen when I normally render `<Legend />` straight away(but then band series are also listed in the legend)

![2024-05-25 12_03_40](https://github.com/recharts/recharts.org/assets/956687/dc700f3e-7a45-4e5c-ba68-79e58086dce8)

Reply:
Hmm using content on Legend shouldn't affect position. I think it's fine to leave legend out for not now

Reply:
I missed to take a look into source for `<Legend>` but finally figured out that returning `<DefaultLegendContent>` from `content` render callback resolves the issue:

![2024-05-25 17_02_20](https://github.com/recharts/recharts.org/assets/956687/9b8e264f-c4f4-454a-b577-3707e319945e)

My initial try was to return `<Legend>` as a content for `<Legend>` so issues were unavoidable 😅